### PR TITLE
Update pyzmq in requirements.txt

### PR DESCRIPTION
**Pull request by ccoulombe**
zmq is really pyzmq

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -11,4 +11,4 @@ torch
 transformers
 ujson
 Werkzeug
-zmq
+pyzmq
```
Comment (loadams) [inline requirements/requirements.txt:14]: Thanks for this change @ccoulombe!  The failed formatting check is looking for the requirements list to be in alphabetical order.  Could you move this up to be below pydantic?

**Review by loadams — APPROVED** [review]

**Review by lekurile — APPROVED** [review]

File: requirements/requirements.txt (hunk 1 above)
Comment (ccoulombe) [reply requirements/requirements.txt:14]: @loadams ah, sure!

Comment (ccoulombe) [reply requirements/requirements.txt]: Done!

Comment (loadams) [reply requirements/requirements.txt]: Thanks!

**Pull request merged**